src/components/SignalCard/card/SignalCard.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import cx from 'classnames'
+import Panel from '@santiment-network/ui/Panel/Panel'
+import { DesktopOnly } from '../../Responsive'
+import MultilineText from '../../MultilineText/MultilineText'
+import SignalCardHeader from './SignalCardHeader'
+import SignalCardBottom from './SignalCardBottom'
+import styles from './SignalCard.module.scss'
+
+const SignalCard = ({
+  id,
+  signal,
+  className,
+  removeSignal,
+  goToSignalSettings,
+  toggleSignal,
+  isUserTheAuthor,
+  deleteEnabled = true,
+  showMoreActions = true,
+  showStatus = true,
+  isMobileVersion = false
+}) => {
+  const isAwaiting = +id <= 0
+  const {
+    title,
+    description = '',
+    isPublic,
+    settings: { type }
+  } = signal
+
+  const clickable = goToSignalSettings && type !== 'trending_words'
+
+  return (
+    <Panel padding className={cx(styles.wrapper, className)}>
+      {isMobileVersion && (
+        <DesktopOnly>
+          <SignalCardHeader
+            deleteEnabled={deleteEnabled}
+            isUserTheAuthor={isUserTheAuthor}
+            isPublic={isPublic}
+            removeSignal={removeSignal}
+            signal={signal}
+          />
+        </DesktopOnly>
```

Comment:
Where we need to use `DesktopOnly` and have `isMobileVersion` at the same time?

Reply:
Do't remember why we are using this feature